src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -81,15 +78,21 @@ class _ComponentStreamStatus:
 @dataclass
 class _BlockingStatus:
     min_duration_sec: float
+    """Minimum blocking duration (in seconds)."""
+
     max_duration_sec: float
+    """Maximum blocking duration (in seconds)."""
 
     def __post_init__(self) -> None:
         assert self.min_duration_sec <= self.max_duration_sec, (
             f"Minimum blocking duration ({self.min_duration_sec}) cannot be greater "
             f"than maximum blocking duration ({self.max_duration_sec})"
         )
         self.last_blocking_duration_sec: float = self.min_duration_sec
+        """Last blocking duration (in seconds)."""  # pylint: disable=pointless-string-statement
```

Comment:
Ouch, weird, I never seen this pylint check triggered before because of this. Do you have any idea why it is triggered here?

Oh, probably because it is in `__post_init__` instead of `__init__`?

Can't we define it in the regular attributes?

Reply:
> Oh, probably because it is in __post_init__ instead of __init__?

yes, that was the reason.

> Can't we define it in the regular attributes?

Yes, we can but with a bit of refactoring as the attributes in __post_init__ are not passed at the class instantiation. I think I can remove one of the attributes that depends on another attribute of the class and for the other one just set a default value.

Reply:
OK, I see. I think the new code is better, and in general, when more advanced uses like these are needed I think it is better to just write a regular class instead.